### PR TITLE
[13.0][IMP] sale_order_secondary_unit: Add menu for secondary UoM

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT28726

Depends on:
- [x] https://github.com/OCA/product-attribute/pull/843

### CODE REVIEW

File: sale_order_secondary_unit/readme/USAGE.rst [hunk 1]
```diff
@@ -5,3 +5,6 @@ To use this module you need to:
 #. Set the conversion factor.
 #. Go to *Sales > Quotation > Create*.
 #. Change quantities in line and secondary unit (produc_qty will be change).
+
+You can manage all Secondary Units of Measure at
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/readme/USAGE.rst]: You can put this in CONFIGURE.rst

File: sale_order_secondary_unit/views/product_secondary_unit_views.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <field name="code" />
+                <field name="name" />
+                <group expand="1" string="Group By">
+                    <filter
+                        string="Product"
+                        name="product_tmpl_id"
+                        context="{'group_by':'product_tmpl_id'}"
+                    />
+                    <filter
+                        string="Secondary UoM"
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/views/product_secondary_unit_views.xml]:
```suggestion
                        string="UoM"
```

File: sale_order_secondary_unit/views/product_secondary_unit_views.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="product_secondary_unit_view_tree" model="ir.ui.view">
+        <field name="name">product.secondary.unit.sale.view.tree</field>
+        <field name="model">product.secondary.unit</field>
+        <field name="mode">primary</field>
+        <field name="priority" eval="20" />
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/views/product_secondary_unit_views.xml]:
```suggestion
        <field name="priority" eval="9999" />
```

File: sale_order_secondary_unit/views/product_secondary_unit_views.xml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                <field name="product_tmpl_id" />
+            </field>
+        </field>
+    </record>
+    <record id="product_secondary_unit_action" model="ir.actions.act_window">
+        <field name="name">Secondary Unit</field>
+        <field name="res_model">product.secondary.unit</field>
+        <field name="view_mode">tree</field>
+    </record>
+    <record id="crm_lead_action_pipeline_view_tree" model="ir.actions.act_window.view">
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/views/product_secondary_unit_views.xml]: This ID is misleading

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: oca_dependencies.txt [hunk 5]
```diff
@@ -1,4 +1,4 @@
-product-attribute
+product-attribute https://github.com/Tecnativa/product-attribute 13.0-imp-product_secondary_unit-add-menu-option
```
Comment (pedrobaeza) [inline oca_dependencies.txt]: You can now remove this

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Review by HaraldPanten — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**